docs/encyclopedia/visibility.mdx

```diff
@@ -156,6 +156,29 @@ order-abracadabra
 
 It does not match `order-~~`.
 
+:::note Filter Composition Quick Reference
+
+**Composition**
+- Data types:
+  - String literals with single or double quotes
+  - Numbers (Integer and Floating Point)
+  - Booleans
+- Comparison: `=`, `!=`, `>`, `>=`, `<`, `<=`
+- Expressions/Operators:
+  - `IN array`
+  - `BETWEEN value AND value`
+  - `STARTS_WITH string`
+  - `IS NULL`, `IS NOT NULL`
+  - `expr AND expr`, `expr OR expr`, `( expr )`
+- Array: `( comma-separated-values )`
+
+**Please note**
+- Wrap attributes with backticks if it contains characters not in
+  `[a-zA-Z0-9]`.
+- `STARTS_WITH` is only available for Keyword search attributes.
```

Comment:
small nit - is keyword supposed to be capitalized?

Reply:
Our capitalizations are not well maintained. In this case, the capitalization is 1. aligned with existing SDK content and 2. indicative that the word refers to a term specific to our framework.

Keeping.

Thank you for raising the question.